### PR TITLE
NOISSUE - Add LoRa route map validation and fix LoRa messages URL

### PR DESCRIPTION
**Pull request by manuio**

### CODE REVIEW

File: lora/redis/streams.go [hunk 1]
```diff
@@ -170,6 +170,9 @@ func (es eventStore) handleUpdateThing(ute updateThingEvent) error {
 	if em.Type != protocol {
 		return errors.New("Lora protocol not found in thing metadatada")
 	}
+	if em.DevEUI != "" {
+		return errors.New("Lora device EUI not found in thing metadatada")
```
Comment (anovakovic01) [inline lora/redis/streams.go]: Please move these errors to `var` block at the beginning of the file.

**Review by anovakovic01 — CHANGES_REQUESTED** [review]

**Review by drasko — DISMISSED** [review]
LGTM

**Review by drasko — DISMISSED** [review]
LGTM

**Review by anovakovic01 — DISMISSED** [review]
LGTM!

**Review by anovakovic01 — DISMISSED** [review]
LGTM!

**Review by drasko — DISMISSED** [review]
LGTM

**Pull request merged**